client/blocks/eligibility-warnings/index.tsx

```diff
@@ -177,6 +178,14 @@ export const EligibilityWarnings = ( {
 			) }
 			<CompactCard>
 				<div className="eligibility-warnings__confirm-buttons">
+					<div className="support-block">
```

Comment:
Inverted the logic to CTA [being always at right](https://github.com/Automattic/wp-calypso/issues/68414#issuecomment-1262263702).